src/main/java/edu/greenblitz/robotName/subsystems/Intake/NeoIntake.java

```diff
@@ -2,12 +2,15 @@
 
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj.DigitalInput;
 
 public class NeoIntake implements IIntake {
 	private CANSparkMax motor;
+	private static DigitalInput beamBreaker;
```

Comment:
why not debouncer? https://docs.wpilib.org/he/stable/docs/software/advanced-controls/filters/debouncer.html